tools/moFF/moff.xml

```diff
@@ -0,0 +1,382 @@
+                    help="Specify the time windows for the matched peak in minutes." />
+                <param argument="--peptide_summary" type="boolean" value="true" label="Output the peptide summary?"/>
+            </when>
+            <when value="mbr">
+                <expand macro="ident_input_macro" allow_multiple="false" input_type="data_collection"/>
+                <param argument="--ext" type="text" value="tab" label="Provide the extension used in the display file name (without the period)"/>
+            </when>
+            <when value="all">
+                <expand macro="ident_input_macro" allow_multiple="false" input_type="data_collection"/>
+                <expand macro="raw_input_macro" allow_multiple="false" input_type="data_collection"/>
+                <param argument="--tol" type="float" value="10" label="Tolerance parameter"
+                    help="Specify the tolerance parameter in ppm." />
+                <param argument="--rt_w" type="float" value="3.0" label="Retention time window"
+                    help="Specify rt window for xic in minutes." />
+                <param argument="--rt_p" type="float" value="1" label="Time window for the peak"
+                    help="Specify the time windows for the peak in minutes." />
+                <param argument="--rt_p_match" type="float" value="1.2" label="Time window for the matched peak"
+                    help="Specify the time windows for the matched peak in minutes." />
+                <param argument="--peptide_summary" type="boolean" value="true" label="Output the peptide summary?"/>
+            </when>
+        </conditional> 
+    </inputs>
+    <outputs>
+        <data format="tabular" name="output_table" label="${tool.name} quantification: ${on_string}">
+            <filter>task['task_selector']=='moff'</filter>
+        </data>
+        <data format="txt" name="output_logs" label="${tool.name} log: ${on_string}">
+            <filter>task['task_selector']=='moff'</filter>
+        </data>
+        <collection name="ident_output" type="list" label="${tool.name} quantification: ${on_string}">
+            <filter>task['task_selector']=='all' or task['task_selector']=='mbr'</filter>
+            <!--discover datasets method -->
+            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.txt" directory="out" format="tabular"/>
+        </collection>
+        <collection name="log_output" type="list" label="${tool.name} logs: ${on_string}">
+            <filter>task['task_selector']=='all' or task['task_selector']=='mbr'</filter>
+            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.log" directory="out" format="txt"/>
+        </collection>
+        <data format="tabular" name="output_peptide_summary" label="${tool.name} peptide summary: ${on_string}">
+            <filter>task['peptide_summary']</filter>
+        </data>
+    </outputs>
+    <tests>
+        <!-- test moff_all -->
```

Comment:
A test with the peptideshaker inputs would be also nice I think.

Reply:
tests 1, 3, and 4 use the PS inputs - only test 2 uses the generic input function.